bot/exts/easter/conversationstarters.py

```diff
@@ -18,9 +28,29 @@ def __init__(self, bot: commands.Bot):
         self.bot = bot
 
     @commands.command()
+    @override_in_channel(all_python_channels)
     async def topic(self, ctx: commands.Context) -> None:
-        """Responds with a random topic to start a conversation."""
-        await ctx.send(random.choice(starters['starters']))
+        """Responds with a random topic to start a conversation, changing depending on channel."""
+        try:
+            # Fetching topics.
+            channel_topics = py_topics[str(ctx.channel.id)]
+
+            if channel_topics:
+                return await ctx.send(random.choice(channel_topics))
```

Comment:
This shouldn't need a `return`.
```suggestion
                await ctx.send(random.choice(channel_topics))
```